script.js

```diff
@@ -0,0 +1,89 @@
+const minSearch = (firstNum, secondNum, thirdNum, fourthNum) => {
+  if (typeof (firstNum && secondNum && thirdNum && fourthNum) === "number") {
+    // console.log(Math.min(firstNum, secondNum, thirdNum, fourthNum));
+    return Math.min(firstNum, secondNum, thirdNum, fourthNum);
+  } else {
+    // console.log("Wrong number of arguments");
+    return "false";
+  }
+};
+minSearch(6, 8, 11, 0);
+
+const gcd = (x, y) => {
```

Comment:
Name is not accurate
```suggestion
const greatestCommonDivider = (x, y) => {
```